frontend/src/pages/scenes/Scene.tsx

```diff
@@ -188,14 +187,14 @@ const SceneComponent: FC<Props> = ({ scene }) => {
             </div>
             <hr />
             <div>
-              <strong className="me-2">Studio URL: </strong>
-              <a
-                href={getUrlByType(scene.urls, "STUDIO")}
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                {getUrlByType(scene.urls, "STUDIO")}
```

Comment:
`getUrlByType` is now unused, consider removing it along with the `URL` type that is defined next to it.